Specs/Scene/ImageryLayerCollectionSpec.js

```diff
@@ -510,7 +511,9 @@ defineSuite([
                     expect(features.length).toBe(2);
                     expect(features[0].name).toEqual('Bar');
                     expect(features[0].description).toContain('Bar!');
+                    expect(features[0].imageryLayer.imageryProvider.tileWidth).toBe(300);
```

Comment:
Oh, I see what you're doing here now (I didn't when I wrote the comment above).  This is clever, but it's probably better to save and check the `ImageryLayer` reference directly instead of checking for particular properties of the imagery provider.